sysutils/pfSense-pkg-LCDproc/files/usr/local/pkg/lcdproc_client.php

```diff
@@ -476,6 +491,15 @@ function outputled_enabled_CFontz633() {
 	}
 }
 
+function cmenu_enabled() {
+	global $config;
+	$lcdproc_config = $config['installedpackages']['lcdproc']['config'][0];
+	$value = $lcdproc_config['controlmenu'];
```

Comment:
It's better to make sure these array keys exist before access them to avoid PHP errors/warnings. Use `empty()` or `isset()` for that